build.gradle

```diff
@@ -75,6 +75,7 @@ configure(javaProjects) {
   // Formatting tasks
   // ================
   googleJavaFormat {
+    toolVersion '1.7'
```

Comment:
1.8 (the default brought by the newer `com.github.sherter.google-java-format`) requires Java 11, so pinning this to 1.7.